### PR TITLE
feat(component): add the Accordion Component

### PR DESCRIPTION
**Pull request by riccardo-vettore**
# What is it?

- [x] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description
The first version of the Accordion Component.

# Use cases
```html
<Accordion>
  <AccordionItem label="Heading 1">
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus
      aliquid architecto delectus deleniti dolor
    </p>
  </AccordionItem>
  <AccordionItem label="Heading 2" disabled>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus
      aliquid architecto delectus deleniti dolor
    </p>
  </AccordionItem>
</Accordion>
```

# Screenshots/Demo

### CODE REVIEW

**Comment by riccardo-vettore** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by riccardo-vettore** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by gioboa — CHANGES_REQUESTED** [review]

**Comment by gioboa** [comment]
Fixes #126

File: packages/headless/src/components/accordion/accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      <button
+        onClick$={(e) => {
+          const target = (e.target as HTMLElement).parentElement;
+          const isTargetOpen = target?.hasAttribute('open');
+          contextService.items.forEach((i: HTMLElement) => {
+            i.removeAttribute('open');
+          });
+          isTargetOpen
+            ? target?.removeAttribute('open')
+            : target?.setAttribute('open', '');
```
[the remark this answers is not in the document]
Comment (riccardo-vettore) [reply packages/headless/src/components/accordion/accordion.tsx:87]: Yes, I fix it.

**Comment by gioboa** [comment]
@all-contributors please add @riccardo-vettore for code

**Review by gioboa — APPROVED** [review]

**Pull request merged**